core/src/main/java/com/backblaze/b2/client/structures/B2GetFileInfoByNameRequest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+package com.backblaze.b2.client.structures;
+
+import java.util.Objects;
+
+public class B2GetFileInfoByNameRequest {
+    private final String bucketName;
+    private final String fileName;
+
+    public B2GetFileInfoByNameRequest(String bucketName,
+                                      String fileName) {
+        // B2Preconditions.checkArg(bucketName != null);
```

Comment:
these are fine preconditions.  how would you feel about uncommenting them?

Reply:
agree. Always good idea do it.